README.md

```diff
@@ -1 +1,346 @@
+
+---
+
+<!-- To generate: gh-md-toc --no-backup --hide-footer README.md -->
+
+**Table of Contents:**
+
+<!--ts-->
+* [OpenShift Helm Charts Repository](#openshift-helm-charts-repository)
+   * [Submitting a Chart](#submitting-a-chart)
+      * [Preparation](#preparation)
+      * [Chart Source](#chart-source)
+      * [Chart Verification Report](#chart-verification-report)
+      * [Chart with Verification Report](#chart-with-verification-report)
+   * [Troubleshooting Pull Request Failures](#troubleshooting-pull-request-failures)
+      * [Error when submitting files not part of any chart](#error-when-submitting-files-not-part-of-any-chart)
+      * [Pull request author is not part of OWNERS file](#pull-request-author-is-not-part-of-owners-file)
+      * [Vendor label mismatch with the directory structure](#vendor-label-mismatch-with-the-directory-structure)
+      * [Chart name mismatch with the directory structure](#chart-name-mismatch-with-the-directory-structure)
+      * [Error when submitting both chart source and tarball](#error-when-submitting-both-chart-source-and-tarball)
+      * [Error when submitting files not related to any chart](#error-when-submitting-files-not-related-to-any-chart)
+      * [Chart name and version mismatch errors](#chart-name-and-version-mismatch-errors)
+      * [Report failures](#report-failures)
+   * [Frequently Asked Questions](#frequently-asked-questions)
+      * [Can I test the pull request in my fork before submitting?](#can-i-test-the-pull-request-in-my-fork-before-submitting)
+      * [Can I use any command-line interface to create pull request?](#can-i-use-any-command-line-interface-to-create-pull-request)
+      * [How to update OWNERS file?](#how-to-update-owners-file)
+   * [Support](#support)
+<!--te-->
+
+---
+
+## Submitting a Chart
+
+### Preparation
+
+To submit a pull request, you need to fork the Git repository and clone it to
+your local system.  If you had followed the [partner connect
+documentation][partners], you should see an _OWNERS_ file under your chart
+directory within your organization directory.
+
+You should see the OWNERS file at:
+
+```
+charts/partners/acme/awesome/OWNERS
+```
+
+For example, if the organization name is `acme` and the chart name is `awesome`.
+The content of the OWNERS file is going to be something like this:
+
+```
+chart:
+  name: awesome
+  shortDescription: A Helm chart for Awesomeness
+publicPgpKey: null
+users:
+  - githubUsername: <username-one>
+  - githubUsername: <username-two>
+vendor:
+  label: acme
+  name: ACME Inc.
+```
+
+The name of the chart that you are submitting must match the value in the OWNERS
+file.  The following section explains how to submit a chart source.  And the
+section followed describes how to submit a chart verification report—finally
+submitting a chart and the report together.
+
+### Chart Source
+
+The first step is to create a directory with the version number as the name. For
```

Comment:
This step is also required if you submit a report only. need to separate it form the submitting a chart procedure.